mgmt_cli_test.py

```diff
@@ -602,6 +603,53 @@ def test_basic_backup(self, ks_names: list = None):
         mgr_cluster.delete()  # remove cluster at the end of the test
         self.log.info('finishing test_basic_backup')
 
+    def test_no_delta_backup_at_disabled_compaction(self):
+        """The purpose of test is to check that delta backup (no changes to DB between backups) takes time -> 0.
+
+        Important test precondition is to disable compaction on all nodes in the cluster.
+        Otherwise, new set of SSTables is created what ends up in the situation that almost no deduplication is applied.
+
+        For more details https://github.com/scylladb/scylla-manager/issues/3936#issuecomment-2277611709
+        """
+        self.log.info('starting test_consecutive_backups')
+
+        self.log.info('Run write stress')
+        self.run_prepare_write_cmd()
+
+        self.log.info('Disable compaction for every node in the cluster')
+        compaction_ops = CompactionOps(cluster=self.db_cluster)
+        for node in self.db_cluster.nodes:
+            compaction_ops.disable_autocompaction_on_ks_cf(node=node)
```

Comment:
I wonder if disable_autocompaction disables also tablets load balancer and tablet split/merge mechanism.

Reply:
That's an interesting question. @denesb do you know the answer?

Reply:
No, disable_autocompaction only disables compactions (and waits for any running one to finish).

Reply:
Note that this only refers to automatic compaction -- user can still start compaction via API (e.g. major, scrub, etc).